gradle/scripts/README.md

```diff
@@ -639,6 +639,32 @@ for more information.
    }
    ```
 
+## Setting a target version with the `java(\\d+)` flag.
+
+By default, setting the `java` flag compiles a module targeting minimum java compatibility with java 8.
```

Comment:
```suggestion
By default, setting the `java` flag compiles a module targeting minimum Java compatibility with Java 8.
```
Please also fix the word in other sentences as well.

Reply:
Thanks, fixed for all public documentation